File: banana.java
// Ellie Kozlowski
// Solution to banana
// 02/10/19

import java.util.Scanner;

public class banana {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		
		// read in input
		int n = in.nextInt();
		String[] arr = new String[n];
		
		for(int i=0; i<n; i++) {
			arr[i] = in.next();
		}
		
		// check to see if the monkey is holding a banana
		for(int i=0; i<n; i++) {
			if(arr[i].equals("banana"))
				System.out.println("He's got the banana!");
			else
				System.out.println("Oh, bummer.");
		}
		
	}
}

File: banana.cpp
// Solution to He's Got the Banana!
// by Jacob Magnuson

#include<bits/stdc++.h>
using namespace std;

int main(){
    string item;
    int T, t;
    cin >> T;
    for(t = 0; t < T; t++) {
        cin >> item;
		if(item == "banana")
			cout << "He's got the banana!" << endl;
		else
			cout << "Oh, bummer." << endl;
    }
    return 0;
}
